lib/gollum/public/gollum/stylesheets/template.scss.erb

```diff
@@ -650,4 +650,8 @@ div.pagination a.disabled {
   .search-context li:nth-child(n+4) {
     display: none;
    }
-}
\ No newline at end of file
+}
+
+.commit .sha {
+  font-family: $font-console;
```

Comment:
Could you check if `Primer.CSS` doesn't already have an alternative way of setting fonts? I really want to minimize `template.scss` and rely on Primer as much as possible. Thanks!

Reply:
Got it. I found `text-mono` class and reverted all the changes in assets.

Reply:
Terrific, thanks!